contexts/utopia-dl/v1.json

```diff
@@ -0,0 +1,8 @@
+{
+  "@context": {
+    "renderMethod": {
```

Comment:
I recommend we put `@protected: true` on all these in case anyone uses them as an example and generally for messaging reasons.

Reply:
Fixed in [b19c857](https://github.com/credential-handler/vc-examples/pull/76/commits/b19c857a8066ae2769c74a640c147eea96cc41b6)